docs/dependencies/metadata-storage.md

```diff
@@ -113,7 +110,7 @@ Note that the format of this blob can and will change from time-to-time.
 ### Rule Table
```

Comment:
The title case style here is not consistent with the changes you made above

Reply:
Fixed, thanks.